tests/polyhedralgrid_test.cc

```diff
@@ -61,27 +60,26 @@ void testGrid( const GridView& gridView )
 
 
         int numIs = 0;
-        IsIt isIt = gridView.ibegin(elem);
-        IsIt isEndIt = gridView.iend(elem);
+        IsIt isIt = gridView.ibegin(*elemIt);
```

Comment:
Since this line apparently works, it cannot just be the dereferencing that makes things go wrong, but using it with `auto`, or perhaps assigning it to a reference. I can only speculate (assuming I do not want to start learning gcc internals)...